cms/static/js/certificates/spec/views/certificate_details_spec.js

```diff
@@ -79,6 +58,20 @@ function(_, Course, CertificatesCollection, CertificateModel, CertificateDetails
         };
 
         beforeEach(function() {
+            window.course = new Course({
```

Comment:
Good question - the outer `beforeEach` was scoped globally; it was getting run before every single jasmine test!